extras/src/main/java/com/google/gson/typeadapters/UtcDateTypeAdapter.java

```diff
@@ -46,16 +46,16 @@ public void write(JsonWriter out, Date date) throws IOException {
   @Override
   public Date read(JsonReader in) throws IOException {
     try {
-      switch (in.peek()) {
-      case NULL:
-        in.nextNull();
-        return null;
-      default:
-        String date = in.nextString();
-        // Instead of using iso8601Format.parse(value), we use Jackson's date parsing
-        // This is because Android doesn't support XXX because it is JDK 1.6
-        return parse(date, new ParsePosition(0));
-      }
+        switch (in.peek()) {
```

Comment:
Could you revert these lines? I'm just seeing indentation changes, and they aren't consistent with the two-space indentation used elsewhere.

Reply:
I'm still seeing changed indentation.